### DISCUSSION TITLE
On mobile each heading shows a '#' sign.

### DISCUSSION THREAD

**Discussion by ghost** [cotes2020/jekyll-theme-chirpy#1879] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

Just as the title says.
See the attached image I snapped from ```chirpy.cotes.page``` below 
![Screenshot_20240727-003930](https://github.com/user-attachments/assets/d3350042-b643-4017-a3b0-fed454a5a9ff)

### Operations you have already tried

Nothing cause I don't understand css that well.
And googling/gpt didn't help.

**Comment by kungfux** [comment]
I assume this works as expected.

The `#` next to each section allows you to copy the link to share.

Since you typically don't have a mouse on mobile, they are always visible so you can use them easily.

**Reply by ghost** [reply]
Is there any way to turn it off?
People who will visit my site won't really be copying anything and it's just bad ui for them

**Reply by kungfux** [reply, marked as answer]
The easiest option is to override the style in `assets/css/jekyll-theme-chirpy.scss` as follows:

```css
.fa-hashtag:before {
  content: none !important;
}
```

**Reply by ghost** [reply]
Thank you it worked ☺️
